### PR TITLE
added refresh button placeholder

### PR DESCRIPTION
**Pull request by omthakare16**
Hey @redxzeta , 
Resolved issue #432 
Added the refresh button placeholder and loading spinner while fetching API data .
Contributed it under Hacktoberfest'22.
Screenshot from 2022-10-02 03-47-55

### CODE REVIEW

File: src/components/home/Home.tsx [hunk 1]
```diff
@@ -54,6 +55,9 @@ const LoadingPetCards = () => {
           <LoadPlaceHolder />
           <LoadPlaceHolder />
         </PetCardFlex>{" "}
+        <Button color="primary">
+          <img className="rotate" src={load} alt="" />
```
Comment (redxzeta) [inline src/components/home/Home.tsx:59]: can you something to the alt tag maybe like "spinner"

**Review by redxzeta — CHANGES_REQUESTED** [review]
lgtm just minor changes

**Comment by omthakare16** [comment]
> lgtm just minor changes

Hey @redxzeta , I have added alt tag as 'spinner' . You can accept and merge the PR now.

File: src/components/home/Home.tsx (hunk 1 above)
Comment (omthakare16) [reply src/components/home/Home.tsx]: Done

**Review by redxzeta — APPROVED** [review]

**Pull request merged**